### PR TITLE
[DDW-1011] fix stake pool list view overlapping news feed

### PR DESCRIPTION
**Pull request by shawnbusuttil**
This PR intends to fix the table header in stake pool list view overlapping the news feed.

Before Change:

Stake Pool List Header Z-Index: 999
News Feed Z-Index: 200

After Change: 

Stake Pool List Header Z-Index: 1
News Feed Z-Index: 200

## Screenshots

| BEFORE  | AFTER |
| ------------- | ------------- |
| Screenshot 2022-03-09 at 15 30 16 | Screenshot 2022-03-09 at 15 29 31 |

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/messages/GGKFXSKC6)

---

### Basics

- [ ] PR assigned to the PR author(s)
- [ ] `input-output-hk/daedalus-dev` and `input-output-hk/daedalus-qa` assigned as PR reviewers
- [ ] If there are UI changes, Alexander Rukin assigned as an additional reviewer
- [ ] All visual regression testing has been reviewed (assign `run Chromatic` label to PR to trigger the run)
- [ ] PR has appropriate labels (`release-vNext`, `feature`/`bug`/`chore`, `WIP`)
- [ ] PR link is added to a Jira ticket, ticket moved to In Review
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes
- [ ] PR contains screenshots (in case of UI changes)
- [ ] CHANGELOG entry has been added to the top of the appropriate section (_Features_, _Fixes_, _Chores_) and is linked to the correct PR on GitHub
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality

- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with typescript types
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing

- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review

- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

File: source/renderer/app/components/staking/stake-pools/StakePoolsTable.scss [hunk 1]
```diff
@@ -105,7 +105,7 @@
       padding: 0 20px;
       position: sticky;
       top: 0;
-      z-index: 999;
+      z-index: 1;
```
Comment (renanvalentin) [inline source/renderer/app/components/staking/stake-pools/StakePoolsTable.scss:108]: Looks good! FYI: i created this file to have a better control over zIndex configuration:
source/renderer/app/themes/mixins/layers.scss

**Review by renanvalentin — APPROVED** [review]

File: source/renderer/app/components/staking/stake-pools/StakePoolsTable.scss (hunk 1 above)
Comment (shawnbusuttil) [reply source/renderer/app/components/staking/stake-pools/StakePoolsTable.scss]: Lets use that!

**Review by miorsufianiohk — APPROVED** [review]
LTGM. Great work @shawnbusuttil . Tested on 21243 👍

File: source/renderer/app/themes/mixins/layers.scss [hunk 2]
```diff
@@ -1 +1,3 @@
 $dialog-z-index: 200;
+$loader-backdrop-z-index: 5;
```
Comment (alexander-rukin) [inline source/renderer/app/themes/mixins/layers.scss:2]: 5 is enough? 

image

Comment (shawnbusuttil) [reply source/renderer/app/themes/mixins/layers.scss:2]: This is something else. But lets fix it too.

The news feed has a 200 z-index. With < 200, that tooltip goes underneath.

Comment (shawnbusuttil) [reply source/renderer/app/themes/mixins/layers.scss:2]: And for your second image: after evaluating and finding that the third party tooltip we are using is set to 9999, I am setting the news feed to a level above that. Are we in agreement that the news feed should be the top most layer in every case?

Comment (alexander-rukin) [reply source/renderer/app/themes/mixins/layers.scss:2]: cc @daniloprates

Comment (renanvalentin) [reply source/renderer/app/themes/mixins/layers.scss:2]: It's also possible to pass a zIndex property to the tooltip component to have a lower value.

**Review by alexander-rukin — APPROVED** [review]
looks ok

**Comment by dmitrii-gaico** [comment]
Do no merge yet as some additional clarifications are required here.

**Comment by shawnbusuttil** [comment]
> Do no merge yet as some additional clarifications are required here.

cc: @alexander-rukin 

What do you think?

https://user-images.githubusercontent.com/27725122/158857980-9eb9f81b-1d34-4783-a636-5fb3880297b1.mov

**Comment by alexander-rukin** [comment]
> > Do no merge yet as some additional clarifications are required here.
> 
> cc: @alexander-rukin
> 
> What do you think?
> 
>  Screen.Recording.2022-03-17.at.17.05.17.mov

It look good! Great job @shawnbusuttil

**Review by dmitrii-gaico — CHANGES_REQUESTED** [review]
@shawnbusuttil Hello. Latest changes looks great however I believe I found one more screen we need to tweak.
The screen is "ITN Screen while sync in progress". If you open news feed and then ITN Screen = the News Feed does not get closed and stay there until you close the ITN. 
ITN Screen when 100% is synced works great btw.
Please see [details attached](https://drive.google.com/file/d/1xql30uZ5ZfG0PZjjpY4j8PgAKHE_f1IP/view?usp=sharing).

**Review by dmitrii-gaico — APPROVED** [review]

**Review by gabriela-ponce — APPROVED** [review]

**Review by ManusMcCole — APPROVED** [review]

**Review by DominikGuzei — APPROVED** [review]
Great work 👍

**Pull request merged**